.env

```diff
@@ -0,0 +1,4 @@
+SENTRY_ORG=justus-soh
```

Comment:
hum ? why is this pushed ?

Reply:
oh noes! This slip through my radar will solve this

Reply:
Done!